packages/cloud-cognitive/src/components/Datagrid/styles/_useNestedRows.scss

```diff
@@ -50,3 +50,21 @@
   background-color: $border-subtle;
   content: '';
 }
+
+.#{$block-class} tr.#{$block-class}__carbon-nested-row {
+  td:first-child:empty,
+  .#{$block-class}__expandable-row-cell {
+    border-bottom: none;
+  }
+}
+
+.#{c4p-settings.$carbon-prefix}--data-table
+  td.#{$block-class}__expandable-row-cell {
+  padding-left: 0;
```

Comment:
I think this should actually use `$spacing-03`.

Using `padding-left: 0`, the button gets shifted a little to the left.

Collapsed | Expanded
-----------|----------
![image](https://user-images.githubusercontent.com/1717398/220521310-f7d513d3-9483-45ce-afc8-1162c78a96ba.png) | ![image](https://user-images.githubusercontent.com/1717398/220521384-baed761d-8347-45ca-b0d7-1499bc969d68.png)